src/popup/components/QrCode.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div id="qr-canvas" />
```

Comment:
In the scope of Vue apps it's considered as a bad pattern to use `id` because this component can be placed more than once in one view.